### ISSUE TITLE
Install Instructions Vs Video

### ISSUE THREAD

**Issue by Jmunsey859** [arismelachroinos/lscript#272]
I hope this will help people that are having troubles. 

Watching the videos vs the instructions on the main page, I'm referencing the Kali install of course. 

The instructions on the main page are missing are miss a step before ./install.sh.

you have to *.sh --- This is very important step for a successful execution of the script. 

another factor for people have trouble is make sure you are in /root directory before the git clone request.
~ use PWD to ensure you are in /root and not in " / "

I also understand the "passwd root" isn't working due to the recent updates. the script still works fine assuming 
you ensure the working directory before the git clone command & 
you add the *.sh before the install.sh 

basically if you follow the video and use PWD the script will install fine. I've installed this script over a hundred time on many systems and architecture.

**Comment by arismelachroinos** [comment]
Thank you for the clarifications. In the Readme I do mention first of all that it is for Kali (1). Then, I do mention that you have to be a root user (2). Then I do mention to type `cd` , which sets the working directory to /root (3). Lastly, I do mention that you have to type `chmod +x install.sh` before you install (4). So are you saying that `chmod +x *.sh` should be added instead of `chmod +x install.sh` ? Or am i missing something?
![image](https://user-images.githubusercontent.com/21193898/99104183-55155080-25e9-11eb-845d-a999347fba0a.png)

**Comment by arismelachroinos** [comment]
On the video I made for the installation, I do mention how to enter the root account for Kali. This is an extra topic I covered that all should already know how to do, at least this is what I assume.
